inst/rmd/qc_report.rmd

```diff
@@ -420,7 +423,83 @@ if(has_processed){
 
 Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
 The raw counts from all remaining cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
-Here we show the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
+The above plot shows the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
+The below plots show the same UMAP embeddings, coloring each cell by the gene expression of the gene indicated in the label for each panel. The genes chosen for plotting are the top 12 highly variable genes identified in the library.
+
+```{r message=FALSE}
+if(has_processed){
+  
+  if(has_var_genes){
+    
+    # select top genes to plot 
+    top_genes <- processed_meta$highly_variable_genes %>%
```

Comment:
A question that may inform later questions... are the hvgs in the metadata in order from most variable to least? If so, we might want to make this an ordered factor to preserve that. If not, we should change so it is, or this might not be the "top" genes.

Reply:
Yes, when I was playing around with this before I checked that they are in order of variance and the first gene that's returned is the one with the highest variation.